### DISCUSSION TITLE
Welcome to cosmonium Discussions!

### DISCUSSION THREAD

**Discussion by el-dee** [cosmonium/cosmonium#23] [Announcements]
## Welcome and thank you for your interest in Cosmonium!

I enabled *Discussions* to provide a space to ask questions about Cosmonium in a more informal way than using issues.

I also plan to make news and announcements posts from time to time.

**Comment by Aztec2012** [comment]
Hi! Very interesting project! But you need much more popilating it. Youtube channel with regular videos and social media will be wery help you.
